build.sh

```diff
@@ -6,11 +6,14 @@ if [ -e ../build/version.sh ]; then
 fi
 IMAGE=kappnav-operator
 
+CURRENT=`pwd`
+PROJECT=`basename "$CURRENT"`
+
 # Update version numbers in yaml files
 ./updateVersionsinYamls.sh
 
 echo "Building ${IMAGE} ${VERSION}"
-docker build --pull --build-arg VERSION=${VERSION} --build-arg BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')  -f build/Dockerfile -t ${IMAGE} .
+operator-sdk build --image-build-args "--pull --build-arg VERSION=${VERSION} --build-arg BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ') --build-arg PROJECT_NAME=${PROJECT}" ${IMAGE}
```

Comment:
Is it possible to run the operator-sdk inside the docker build?  What would be the pro/con of doing that?

Reply:
How big and heavy is the SDK?

Reply:
The v0.10.0 binary looks to be about 64 MBs: https://github.com/operator-framework/operator-sdk/releases/download/v0.10.0/operator-sdk-v0.10.0-x86_64-linux-gnu.

Reply:
The operator-sdk build directly produces a Docker image (or Podman or Buildah image). Doesn't look like they created anything that makes it easy to build inside of a container.

Reply:
There's an install script (.travis/install-operator-sdk.sh) that's used by the Travis build. I didn't advertise it in the README, though I suppose developers could use it too. Apparently only took 1.6 seconds to run in the last Travis build.